media-video/ffmpeg/ffmpeg-4.0.ebuild

```diff
@@ -347,14 +348,19 @@ multilib_src_configure() {
 		myconf+=( $(use_enable ${i%:*} ${i#*:}) )
 	done
 
-	# Incompatible features: openssl and gnutls
-	# openssl support provides a (strict) superset of gnutls support as of 2017.11.30
+	# Incompatible features: openssl or libressl and gnutls
+	# openssl and libressl support provides a (strict) superset of gnutls support as of 2017.11.30
 	# So, we warn the user and disable gnutls
 	if use openssl && use gnutls; then
 		ewarn "openssl and gnutls are mutually exclusive in ${PN}, disabling gnutls since openssl provides more features"
 		myconf+=( --disable-gnutls )
 	fi
 
+	if use libressl && use gnutls; then
```

Comment:
nit: if use openssl && use libressl ; then disable libressl

(I prefer not to rely on liberssl & openssl being mutual blockers as that will make configure bail out here)

Reply:
Portage produces the warning before the ebuild can even get to configure... the above code meets the requirement for the user to set the conditions how they wish as is.  To remove the mutual block would break packages tree wide.  It would also cause a lot of confusion.

I will update -9999 for your review once all of this is solidified.